dffml/source/csv.py

```diff
@@ -59,9 +59,14 @@ class CSVSource(FileSource, MemorySource):
                         }
                     }
                 )
-            # Create the repo with the source URL being the row index
-            repo = Repo(str(i), data=repo_data)
-            i += 1
+            if self.config.src_url == key:
```

Comment:
We'll change the default for `config.src_url` to `None`, and then check if it was set by checking if it's not None

```python
if self.config.src_url is not None and self.config.src_url == key:
```

Reply:
Done :+1: